### ISSUE TITLE
How to customize default layout

### ISSUE THREAD

**Issue by anobilisgorse** [alamkanak/Android-Week-View#154]
Upon encountering your library, I was glad that there exists one that matches with my concept design for a scheduler. 

![class schedule](https://cloud.githubusercontent.com/assets/8383878/8692522/6473e98a-2b01-11e5-8eb1-ec65ea39397c.jpg)

Sadly, due to my aesthetics, I can't seem to tolerate the little difference it makes with my concept design. As you can see from the image above, my concept is much more similar to the native DayView Calendar app of Android:
![screenshot_2015-07-15-12-28-30](https://cloud.githubusercontent.com/assets/8383878/8692542/a90bbcee-2b01-11e5-9cc3-19fe62cbf5be.png)

So I was wondering if there's a workaround to tweak or customize your default layout regarding the alignment, positioning, and customization of the time at the left side(with respect to my concept design above), and also that little padding/margin(?) between the calendar itself and the header
(as seen here):
![screenshot_2015-07-15-14-45-47](https://cloud.githubusercontent.com/assets/8383878/8692551/d6433890-2b01-11e5-8544-d275f7ad69d6.png)

That is all, hoping you would reply ASAP.

**Comment by caske33** [comment]
I'd suggest that you take a look at the different [(color) attributes](https://github.com/alamkanak/Android-Week-View/blob/master/library/src/main/res/values/attrs.xml) you can tweak on the library.

**Comment by anobilisgorse** [comment]
@caske33 I'm sorry but it's not the color that I would like to tweak, instead, the one encircled in the image, that little "offset" from the header is what I would like to remove.

**Comment by caske33** [comment]
I'm not 100% sure if there's an attribute for that yet. Have you tried all the dimensions attributes too?

**Comment by anobilisgorse** [comment]
Yes I have. I was speculating it has something to do with "headerColumnPadding" or anything that has a padding with it but there's no bulge in the layout.

Is there no other way to tweak the Java source codes directly?

**Comment by caske33** [comment]
You could import this project as a git and gradle submodule and then you could edit the source code directly. Or you can make a pull request with a new attribute.

**Comment by anobilisgorse** [comment]
I see. But once I'm done editing the source code, how should I import/integrate the now edited library to my project? Cause I just used a gradle dependency for using this third library.

**Comment by caske33** [comment]
`compile project(':weekview:library')` (if weekview is in your root library. Next to the "app" module).

**Comment by anobilisgorse** [comment]
Thanks. I think I'll try my own luck in tweaking the source code.

**Comment by lolobosse** [comment]
It is possible to remove that, I made it in the app of my employer (Shore.com, however you cannot access to the result without an paying account). I do not know if it's the final solution, but I wrote some code to do that: lolobosse/Android-Week-View@70b010b

**Comment by obaslan** [comment]
I believe this is one of the biggest issues regarding the interface at the moment. Does anyone have any ideas if this can be solved easily?
